app/src/androidTest/java/com/duckduckgo/app/systemsearch/SystemSearchViewModelTest.kt

```diff
@@ -110,26 +105,28 @@ class SystemSearchViewModelTest {
         assertFalse(viewState.expanded)
     }
 
+    @Ignore
     @Test
-    fun whenDatabaseIsSlowThenIntroducingTextDoesNotCrashTheApp() = coroutineRule.runBlocking {
-        (coroutineRule.testDispatcherProvider.io() as TestCoroutineDispatcher).pauseDispatcher()
-        testee =
-            SystemSearchViewModel(givenEmptyUserStageStore(), mockAutoComplete, mockDeviceAppLookup, mockPixel, mockFavoritesRepository, mockFaviconManager, mockSettingsStore, coroutineRule.testDispatcherProvider)
+    fun whenDatabaseIsSlowThenIntroducingTextDoesNotCrashTheApp() = runTest {
```

Comment:
@CDRussell is safe to remove this test. The ViewModel logic since then has changed and the scenario does not exist anymore.

FYI: I think this would have been the code to use a paused dispatcher:

```
    @Test
    fun whenDatabaseIsSlowThenIntroducingTextDoesNotCrashTheApp() {
        val pausedTestDispatcher: TestDispatcher = StandardTestDispatcher()

        val testDispatcherProvider = object : DispatcherProvider {
            override fun default(): CoroutineDispatcher = pausedTestDispatcher
            override fun io(): CoroutineDispatcher = pausedTestDispatcher
            override fun main(): CoroutineDispatcher = pausedTestDispatcher
            override fun unconfined(): CoroutineDispatcher = pausedTestDispatcher
        }

        runTest(pausedTestDispatcher) {
            testee = SystemSearchViewModel(
                givenEmptyUserStageStore(),
                mockAutoComplete,
                mockDeviceAppLookup,
                mockPixel,
                mockFavoritesRepository,
                mockFaviconManager,
                mockSettingsStore,
                testDispatcherProvider
            )
            testee.resetViewState()
            testee.userUpdatedQuery("test")

            // no crash
        }
    }
```